module/VuFind/src/VuFind/Auth/AbstractBase.php

```diff
@@ -304,6 +304,19 @@ public function supportsPasswordRecovery()
         return false;
     }
 
+    /**
+     * Does this authentication method support connecting library card of
+     * currently authenticated user?
+     *
+     * @return bool
+     */
+    public function supportsConnectingLibraryCard()
+    {
+        // By default, connecting library card of currently authenticated user
+        // is not supported.
+        return false;
```

Comment:
How about changing this to `return method_exists($this, 'connectLibraryCard');`? That way, by adding the required method, the supports method will automatically return true, and there's less work to implement support. It's still possible to extend the method if additional logic is needed, but it makes the default implementation more flexible.